Google.PowerShell/BigQuery/GcbqCmdlet.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright 2015-2016 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Bigquery.v2;
+using Google.PowerShell.Common;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// Base class for Google Cloud PubSub cmdlets.
```

Comment:
Just noticed there's a problem with the comments there.  Fixed that.  

Was your comment talking about the using statement?